.travis.yml

```diff
@@ -15,5 +15,7 @@ before_script:
   - php -r "printf('PHP %s', phpversion());"
   - composer self-update
   - composer install --no-interaction
+  - composer dumpautoload -o
```

Comment:
Why not merge that with the previous command ? The autoload is already generated on the `install`. Adding the `-o` instruction would optimize it. 

```
composer install --no-interaction -o
```